From 53493b95fc87fe1c0e13c084dc434201c658c5db Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Thu, 6 Sep 2018 12:52:16 +0900
Subject: [PATCH 5/8] tools/sppc: update app launcer for custom SPP repo

To launch executable in custom repo, change the path from absolute
path to just filename. Path of the file is defined as PATH in each
of Dockerfiles and no need to use absolute path by previous change.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 tools/sppc/app/spp-nfv.py     | 5 +----
 tools/sppc/app/spp-primary.py | 5 +----
 tools/sppc/app/spp-vm.py      | 5 +----
 3 files changed, 3 insertions(+), 12 deletions(-)

diff --git a/tools/sppc/app/spp-nfv.py b/tools/sppc/app/spp-nfv.py
index cdb61a8..6506fed 100755
--- a/tools/sppc/app/spp-nfv.py
+++ b/tools/sppc/app/spp-nfv.py
@@ -79,10 +79,7 @@ def main():
         docker_run_opt = '-it'
 
     # Setup spp_nfv command.
-    cmd_path = '%s/../spp/src/nfv/%s/spp_nfv' % (
-        env.RTE_SDK, env.RTE_TARGET)
-
-    spp_cmd = [cmd_path, '\\']
+    spp_cmd = ['spp_nfv', '\\']
 
     # Do not use 'app_helper.setup_eal_opts()' because spp_nfv does
     # not use virtio.
diff --git a/tools/sppc/app/spp-primary.py b/tools/sppc/app/spp-primary.py
index a3fc9f8..6a004f6 100755
--- a/tools/sppc/app/spp-primary.py
+++ b/tools/sppc/app/spp-primary.py
@@ -106,10 +106,7 @@ def main():
         container_image, '\\']
 
     # Setup spp primary command.
-    cmd_path = '%s/../spp/src/primary/%s/spp_primary' % (
-        env.RTE_SDK, env.RTE_TARGET)
-
-    spp_cmd = [cmd_path, '\\']
+    spp_cmd = ['spp_primary', '\\']
 
     # Do not use 'app_helper.setup_eal_opts()' because spp_primary does
     # not use virtio vdev but TAP or vhost, which should be added manually.
diff --git a/tools/sppc/app/spp-vm.py b/tools/sppc/app/spp-vm.py
index 864ec61..d590253 100755
--- a/tools/sppc/app/spp-vm.py
+++ b/tools/sppc/app/spp-vm.py
@@ -75,10 +75,7 @@ def main():
         common.error_exit('SPP_CTRL_IP')
 
     # Setup spp_vm command.
-    cmd_path = '%s/../spp/src/vm/%s/spp_vm' % (
-        env.RTE_SDK, env.RTE_TARGET)
-
-    spp_cmd = [cmd_path, '\\']
+    spp_cmd = ['spp_vm', '\\']
 
     file_prefix = 'spp-l2fwd-container%d' % dev_ids_list[0]
     eal_opts = app_helper.setup_eal_opts(args, file_prefix)
-- 
2.7.4

